pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -284,6 +287,25 @@ func (opt *Options) Run(suite *TestSuite, junitSuiteName string) error {
 		return err
 	}
 
+	cl, err := configv1client.NewForConfig(restConfig)
+	if err != nil {
+		return fmt.Errorf("failed to create client for CVO: %v", err)
+	}
+
+	cv, err := cl.ConfigV1().ClusterVersions().Get(ctx, "version", metav1.GetOptions{})
+	if err != nil {
+		return fmt.Errorf("failed to get cvo version object from cluster: %v", err)
+	}
+
+	storageEnabled := true
+	if cluster.KnowsCapability(cv, "Storage") {
+		storageEnabled = cluster.HasCapability(cv, "Storage")
+	}
+
+	if !storageEnabled {
+		fmt.Fprintln(opt.Out, "Skipping [sig-storage] tests because Storage capability disabled")
```

Comment:
100% of sig-storage tests can't be run when this cap is disabled?  including things like emptydir storage?

Reply:
nit, can we move this line down to an `else` block of the newly-conditional `tests = append(tests, storageTestsCopy...)`?

```go
if storageEnabled {
  // I thought about randomizing the order of the kube, storage, and openshift tests, but storage dominates our e2e runs, so it doesn't help much.
  storageTestsCopy := copyTests(storageTests)
  q.Execute(testCtx, storageTestsCopy, max(1, parallelism/2), status.Run) // storage tests only run at half the parallelism, so we can avoid cloud provider quota problems.
  tests = append(tests, storageTestsCopy...)
} else {
  fmt.Fprintln(opt.Out, "Skipping [sig-storage] tests because Storage capability disabled")
}
```

Keeping that collected together reduces the chances that someone touches one of the branches without noticing that they may also need to adjust the other.